lib/mayaUsd/fileio/translators/translatorMayaReference.cpp

```diff
@@ -38,10 +38,8 @@
 #include <maya/MDGModifier.h>
 #include <maya/MFileIO.h>
 #include <maya/MFileObject.h>
-#include <maya/MFnCamera.h>
```

Comment:
Removed useless includes while I was at it (MFnTransform and MFnCamera for Maya reference translation?)